File: csharp/Algorithms.Tests/Ex41Test.cs
using Xunit;

namespace Algorithms.Tests
{
    public class Ex41Test
    {
        [Fact]
        public void Test()
        {
            int[] array = { 2, 4, 6 };
            Assert.Equal(12, Ex41.iterativeSum(array));
            Assert.Equal(12, Ex41.recursiveSum(array));
        }
    }
}

File: java/src/test/java/grokking/algorithms/Ex41Test.java
package grokking.algorithms;

import org.junit.Test;

public class Ex41Test {
    @Test
    public void test() {
        int[] array = { 2, 4, 6 };
        org.junit.Assert.assertEquals(12, Ex41.iterativeSum(array));
        org.junit.Assert.assertEquals(12, Ex41.recursiveSum(array));
    }
}